SwiftMailer/MailjetTransport.php

```diff
@@ -185,23 +189,13 @@ public function bulkSend(array $messages, &$failedRecipients = null) {
             }
         }
         // Create mailjetClient
-        $mailjetClient = $this->createMailjetClient();
-
+        $this->createMailjetClient();
```

Comment:
This line has no effect at the moment. We create new client instance and throw it into space.

I suggest we move the creation of the client before the foreach loop and keep the `is_null` condition.

```php
// Create mailjetClient
if (is_null($this->mailjetClient)) {
    $this->mailjetClient = $this->createMailjetClient();
}
foreach ($messages as $message) {
    // ...
}
```

This will have performance gain as well.